### PR TITLE
Actions menu IA changes - review required

### PR DESCRIPTION
**Pull request by chri2547**
@araiford , actions menu is final review and merge!

image

### CODE REVIEW

File: _component_design/actions-menu.md [hunk 1]
```diff
@@ -18,27 +17,45 @@ helix-ui-javascript: true
...
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Best practices
+
+Consider the following best practices when you construct an actions menu.
+
+- Items in the actions menu must initiate an action. Do not use an actions menu for navigation.
```
Comment (araiford) [inline _component_design/actions-menu.md]: Maybe a tweak?
`Items in the actions menu must only initiate processes for changing settings or system objects.  Do not use items in the actions menu for navigating to a different location.`

File: _component_design/actions-menu.md [hunk 2]
```diff
@@ -18,27 +17,45 @@ helix-ui-javascript: true
...
+- If an action name is longer than the length of the button, extend the menu beyond the right side of the button.
+- If an action label is too long, truncate the text and display the entire name when the users hover over the label.
+
```
Comment (araiford) [inline _component_design/actions-menu.md]: `Avoid lengthy action labels.  Keep them to three short words or less.`

File: _component_design/actions-menu.md [hunk 3]
```diff
@@ -80,24 +114,45 @@ When using section headers within an actions menu, consider the following practi
...
 
-There are 8px of space between each section.
+{% endcolumn %}
 
-Minimum menu width is 10 rem.  Maximum menu width is 20 rem.  
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
 
-If an action name is longer than the length of the button the menu can extend beyond the right side of the button.
+{% figure [caption:"Actions menu style"] [class:"image bg-light border"] %}
+<embed src="{{site.url}}/assets/images/components/inputs-and-controls/actions-menu/actions-menu-menu-specifications.png" width="541"/>
```
Comment (araiford) [inline _component_design/actions-menu.md]: this needs to be actions-menu-style.png

File: _component_design/actions-menu.md [hunk 4]
```diff
@@ -18,27 +17,45 @@ helix-ui-javascript: true
...
 
-### When to use
+## When to use
 
-- Use actions menus to consolidate multiple related actions into a single menu.
+An actions menu consolidates multiple related actions into a single menu. Use an actions menu in the following situations:
 
-### When not to use
+  - When a user takes action on the current page or in a [modal]({{site.baseurl}}/components/modals.html)
+  - The user creates or edits items on a different page
```
[the remark this answers is not in the document]
Comment (chri2547) [reply _component_design/actions-menu.md]: I'm not sure what that bullet point means. I'll go with your tweak!

File: _component_design/actions-menu.md [hunk 5]
```diff
@@ -18,27 +17,45 @@ helix-ui-javascript: true
...
+Consider the following best practices when you construct an actions menu.
+
+- Items in the actions menu must initiate an action. Do not use an actions menu for navigation.
+- If an action name is longer than the length of the button, extend the menu beyond the right side of the button.
+- If an action label is too long, truncate the text and display the entire name when the users hover over the label.
+
+Consider the following best practices when you construct an actions menu that contains many items.
+
+- Group related items into headers. Headers provide clarity and help users locate menu items of interest.
+- Use a header when the actions menu includes more than three actions. Do **not** use a header when the actions menu includes three or fewer actions.
```
Comment (stephamon) [inline _component_design/actions-menu.md]: "Use a header only when the actions menu includes more than three actions."

The "do not" sentence is redundant.

File: _component_design/actions-menu.md [hunk 6]
```diff
@@ -52,21 +69,38 @@ helix-ui-javascript: true
 
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
 
-### Best practices
+## Specifications
+
+Use the following specifications when you construct an actions menu.
```
Comment (stephamon) [inline _component_design/actions-menu.md]: End with a colon here instead of a period.

File: _component_design/actions-menu.md [hunk 7]
```diff
@@ -18,27 +17,45 @@ helix-ui-javascript: true
...
+
+## Best practices
+
+Consider the following best practices when you construct an actions menu.
+
+- Items in the actions menu must initiate an action. Do not use an actions menu for navigation.
+- If an action name is longer than the length of the button, extend the menu beyond the right side of the button.
+- If an action label is too long, truncate the text and display the entire name when the users hover over the label.
+
+Consider the following best practices when you construct an actions menu that contains many items.
```
Comment (stephamon) [inline _component_design/actions-menu.md]: End with a colon here instead of a period.

File: _component_design/actions-menu.md [hunk 8]
```diff
@@ -80,24 +114,45 @@ When using section headers within an actions menu, consider the following practi
 
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-### Specifications
+### Style
 
-Section headers in the menu are optional. They are used if clarity is needed, or if there are more than seven actions in the list.
+Use these specifications when you construct an actions menu.
```
Comment (stephamon) [inline _component_design/actions-menu.md]: "Use the following specifications when you construct an actions menu:"

**Pull request merged**